### ISSUE TITLE
chrome.webRequest: 1st example produces warning

### ISSUE THREAD

**Issue by NV** [GoogleChrome/developer.chrome.com#3893]
https://developer.chrome.com/docs/extensions/reference/webRequest/#manifest

```
  "permissions": [
    "webRequest",
    "*://*.google.com/"
  ],
```

In the extensions console, I see this warning: 

    Permission '*://*.google.com/' is unknown or URL pattern is malformed.

**Comment by jpmedley** [comment]
Sorry for the delay. Are you working in Manifest V2 or Manifest V3?

**Comment by NV** [comment]
Manifest V3.

**Comment by jpmedley** [comment]
Web requests are done [`declarativeNetRequest` in MV3](https://developer.chrome.com/docs/extensions/mv3/mv3-migration/#modifying-network-requests). (That material [links to this](https://developer.chrome.com/docs/extensions/reference/declarativeNetRequest/).)

I'm new to extensions and still trying to sort this myself. If you have questions, I'll help you find answers. Long term, I need to be able to answer them. Short term, I need to improve the materials about `declarativeNetRequest`.

**Comment by NV** [comment]
Examples on [declarativeNetRequest](https://developer.chrome.com/docs/extensions/reference/declarativeNetRequest/) are also for MV2 and produce the exact same error in MV3:
```
Permission '*://*.google.com/' is unknown or URL pattern is malformed.
```

I suggest updating the example code to include the correct [host_permissions](https://developer.chrome.com/docs/extensions/mv3/mv3-migration/#host-permissions) for MV3. You already have examples for both MV2 and MV3 on some documentation pages like [tabs](https://developer.chrome.com/docs/extensions/reference/tabs/#get-the-current-tab).

**Comment by AmySteam** [comment]
Fixed #6941 and #6921
